modules/@apostrophecms/login/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extendMethods(self) {
+    return {
+      getBrowserData(_super, req) {
+        const data = _super(req);
+        if (data.user) {
+          data.user.role = req.user.role;
```

Comment:
We don't expose the logged-in user's role on the front end by default, so add that information.